test/definition/component-info.test.ts

```diff
@@ -43,6 +43,12 @@ describe('ComponentInfo', () => {
                                 type: 'editable',
                             },
                         ],
+                        properties: [
```

Comment:
I guess you could also opt for a structure like:
```typescript
{
    selectProperty: {
        dataType: 'styles',
    }
}
```

Reply:
Yeah, was already doubting what would be better. But I guess this indeed better aligns with the way the components are listed here as well (and allows for quick lookups).

Reply:
It depends on our goal I think. The order of the properties is also the order it is appearing in the component properties panel.

Reply:
I guess that's only important when you want to build a properties UI? In that case you need the full definition anyway. My assumption is that the component info is used by article processing tools to correctly interpret the article, allowing conversion / data lookup.

Reply:
Indeed 👍